scripts/install

```diff
@@ -16,4 +16,4 @@ else
 fi
 
 "$PIP" install -r "$REQUIREMENTS"
-"$PIP" install -e .
+"$PIP" install .
```

Comment:
Isn't this the only change needed? 🤔

Reply:
@Kludex For me, the `scripts/test` only fully passed after also adding the two lines in the `requirements.txt`